### PR TITLE
GFM-104 - refactored address lookup to remove duplicate code

### PR DESCRIPTION
**Pull request by joefitter**
* renamed 'address' to 'address-lookup'
* removed address-inside, address-outside and no-postcode
* replaced the above with one step - 'address'
* no-postcode controller now called `address`
* added `isDomestic` to address controller locals - true if UK
* added conditional panel to address - shows postcode if domestic
* abstracted edit-postcode html to partial - used by address.html and address-lookup.html
* bugfix - unset `addresses` and apiErrorMessage from sessionModel on BT postcode and success api call

### CODE REVIEW

File: apps/gro/fields/address.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 module.exports = {
-  address: {
-    className: ['address']
+  'address-textarea': {
+    validate: ['required'],
+    'ignore-defaults': true,
+    formatter: ['trim', 'hyphens']
```
Comment (JoeChapman) [inline apps/gro/fields/address.js]: if one specifies a list of formatters, is it necessary to also ignore the others?

Comment (joefitter) [inline apps/gro/fields/address.js]: yes

Comment (joefitter) [inline apps/gro/fields/address.js]: https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/formatting/index.js#L21 does default here, then additional underneath. This isn't part of this PR though, just renamed the file.

**Comment by joefitter** [comment]
@gxxm @bendiggle @JoeChapman 😍  please :)

**Comment by JoeChapman** [comment]
This LGTM

**Pull request merged**